aa-plugin-dev/.github/workflows/test.yml

```diff
@@ -0,0 +1,64 @@
+name: test
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
~Not sure if we want to update this now, but if we do, should this use `main` instead of `master`?~

**Edit**: Seen your comment about no other commits on top of template just yet, so assuming changes like this would come in a subsequent PR